cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/CommonConstants.java

```diff
@@ -91,8 +91,11 @@ private CommonConstants() {
             + "# \n"
             + "# For those usages not covered by the GNU Affero General Public License please "
             + "contact with iot_support at tid dot es";
-    
+    /**
+     * 
+     *
+     */
     // Enumeration for ManagementInterface
-    public enum LoggingLevels { FATAL, ERROR, WARN, INFO, DEBUG, ALL, OFF }
+    public enum LoggingLevels { FATAL, ERROR, WARN, INFO, DEBUG, TRACE, ALL, OFF }
```

Comment:
Is this new TRACE level really necessary? Why don't reuse DEBUG for it?

Reply:
Done